docs/development.md

```diff
@@ -1,78 +1,126 @@
 # AutoRest PowerShell Generator - Development
```

Comment:
Please avoid using AI to refactor our documentation, as it complicates tracking necessary changes. Instead, add a separate section or markdown file for the dev container.

Reply:
Changed